### PR TITLE
Increase Combat Units Cost and Tweak AI

### PR DESCRIPTION
**Pull request by WilliamWeberBerrutti**
Motivations:

- Aids on performance issues when there's a lot of units in the match
- By reducing the combat units amount, it's easier to micro, being a little more important for the game
- Tweak AI with the new cost changes and some other changes like building at least 2 Storages so it's harder to kill it

The cost increase proportion is +100%. Units not affected are: Miner, Builders, Recon Tank, Money Transport

### CODE REVIEW

**Review by OcelotWalrus — APPROVED** [review]
The code's good and the idea too. @Rampoina Cyantusk and me playtetsed it.

File: mods/hv/rules/bots.yaml [hunk 1]
```diff
@@ -64,39 +72,47 @@ Player:
...
+			field: 12000
+			silo: 13000
+			uplink: 13000
 	BuildingRepairBotModule:
 		RequiresCondition: enable-rogue-ai
 	SquadManagerBotModule@RogueAI:
 		RequiresCondition: enable-rogue-ai
 		SquadSize: 20
-		ExcludeFromSquadsTypes: miner, builder, builder2, tanker1, tanker2, minelayer, ecmtank, technician, broker, mineship, mineship2, blaster
+		ExcludeFromSquadsTypes: miner, builder, builder2, tanker1, tanker2, minelayer, technician, broker, mineship, mineship2, blaster, flamer
```
Comment (Mailaender) [inline mods/hv/rules/bots.yaml:107]: The flamer is a regular attack unit not handled by other bot modules.

Comment (WilliamWeberBerrutti) [reply mods/hv/rules/bots.yaml:107]: So the idea is to remove flamer there? So the Flamer Pod is going to be included on squads? It shouldn't, in my opinion.

Comment (Mailaender) [reply mods/hv/rules/bots.yaml:107]: Then write a bot module for them, or they will stay in base.

File: mods/hv/rules/bots.yaml [hunk 2]
```diff
@@ -16,9 +16,15 @@ Player:
...
-		AdditionalMinimumRefineryCount: 0
-		NewProductionCashThreshold: 10000
+		MaximumExcessPower: 500
+		InititalMinimumRefineryCount: 1
+		AdditionalMinimumRefineryCount: 1
+		PlaceDefenseTowardsEnemyChance: 0
+		StructureProductionInactiveDelay: 0
+		StructureProductionActiveDelay: 5
+		StructureProductionRandomBonusDelay: 0
+		StructureProductionResumeDelay: 5
```
Comment (Mailaender) [inline mods/hv/rules/bots.yaml:26]: These are typically added for performance reasons so not all bots compute at the same time. Please don't set it to 0 or very low values.

Comment (WilliamWeberBerrutti) [reply mods/hv/rules/bots.yaml:26]: What values do you suggest?

Comment (Mailaender) [reply mods/hv/rules/bots.yaml:26]: Just remove them. They are arbitrary.

**Pull request merged**